README.md

```diff
@@ -12,8 +12,6 @@ libraries:
   and clients in the .NET environment;
 * log4net (v2.0.12) - a library providing logging services for purposes
   of application debugging and auditing;

```

Comment:
Remove also `toolset v142 for C++` further down in the README, it's not needed for anything else.